roles/ovirt-storages/tasks/main.yml

```diff
@@ -16,6 +16,7 @@
     posixfs: "{{ item.value.posixfs | default(omit) }}"
     glusterfs: "{{ item.value.glusterfs | default(omit) }}"
     fcp: "{{ item.value.fcp | default(omit) }}"
+    override_luns: "{{ item.value.override_luns | default(omit) }}"
```

Comment:
I think it's part only of iscsi, no? Also please document it in `README.md`. Thanks!